src/dotnet/APIView/APIView/AttributeAPIV.cs

```diff
@@ -34,10 +35,10 @@ public AttributeAPIV(AttributeData attributeData)
 
         public override string ToString()
         {
-            var returnString = new StringBuilder();
             var renderer = new TextRendererAPIV();
-            renderer.Render(this, returnString);
-            return returnString.ToString();
+            var list = new List<LineAPIV>();
+            renderer.Render(this, list);
+            return list.First().DisplayString;
```

Comment:
Use StringListAPIV here too

Reply:
You can add ToString(bool noTrailingNewline) overload to avoid string.Replace in tests too.

Reply:
Would that mean adding a `ToString(bool noTrailingNewline)` overload to each class, in addition to `StringListAPIV`?

Reply:
You know you don't need to overload, just don't append the last newline and you don't need to Replace in tests.